docker/Dockerfile

```diff
@@ -53,27 +53,19 @@ RUN curl -fSL ${MIRROR}/${NIFI_BINARY_URL} -o ${NIFI_BASE_DIR}/nifi-${NIFI_VERSI
     && echo "$(curl ${MIRROR}/${NIFI_TOOLKIT_URL}.sha256) *${NIFI_BASE_DIR}/nifi-toolkit-${NIFI_VERSION}-bin.tar.gz" | sha256sum -c - \
     && tar -xvzf ${NIFI_BASE_DIR}/nifi-toolkit-${NIFI_VERSION}-bin.tar.gz -C ${NIFI_BASE_DIR} \
     && rm ${NIFI_BASE_DIR}/nifi-toolkit-${NIFI_VERSION}-bin.tar.gz \
-    && chown -R nifi:nifi ${NIFI_HOME}
-
-# Establish best practices for running NiFi
-RUN echo "*  hard  nofile  50000" >> /etc/security/limits.conf \
-    && echo "*  soft  nofile  50000" >> /etc/security/limits.conf \
-    && echo "*  hard  nproc  10000" >> /etc/security/limits.conf \
-    && echo "*  soft  nproc  10000" >> /etc/security/limits.conf \
-    && echo "vm.swappiness = 0" >> /etc/sysctl.conf \
-    && sysctl -w net.ipv4.netfilter.ip_conntrack_tcp_timeout_time_wait="1" \
-    && sysctl -w net.ipv4.ip_local_port_range="10000 65000"
+    && chown -R nifi:nifi ${NIFI_HOME} ${NIFI_TOOLKIT_HOME}
 
 # Web HTTP(s) & Socket Site-to-Site Ports
 EXPOSE 8080 8443 10000
 
+USER nifi
+
 COPY ./files/nifi-env.sh $NIFI_HOME/bin/nifi-env.sh
 
 COPY ./files/docker-entrypoint.sh /
```

Comment:
Believe we need to remove this line as well.

Reply:
yes and I will do a diff with the official one to see how much we diverged